careers/careers/utils.py

```diff
@@ -14,10 +14,10 @@ def order_by_function(x):
         order_by = order_by_function
 
     return (
-        sorted(set(
+        sorted(
```

Comment:
And now with more caching using `ADD`. See https://github.com/docker/docker/pull/12031